### PR TITLE
Render termsheet

### PR DESCRIPTION
**Pull request by johnjudeh**
This work is to create a simple command line tool for potential users of the Origin termsheet template and the Airbrush data standard to be able to programmatically create termsheets. It gives them everything needed to also use with any other templates that rely on the same data.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +16,54 @@ The enclosed .yaml file leverages the OpenApi Specification, enabling consumers
...
+pip install -r requirements.txt
+```
+
+This should download all the requirements you need to run the command line tool. You are now ready to use
+the tool.
+
+### Usage
+
+To make it easy to test out Origin's termsheet template, we have created a few example trades that you can
+try out. You can find these in the [Examples folder](./examples/). You willd find both an
```
Comment (uttamo) [inline README.md]: typo in "willd"

File: create_termsheet.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        ext: The file extenion to validate the file path against. In the format '.*'
+        must_exist: Boolean for whether to check existence of file in validation. Defaults to True
+
+    Returns:
+        str | None: The string passed to it if its valid, otherwise None
+    """
+    path = Path(file_path)
+    if must_exist and not (path.exists() and path.is_file):
+        parser.error(f"The file {file_path} does not exist")
+    elif not path.suffix == ext:
```
Comment (uttamo) [inline create_termsheet.py]: think the standard way of doing this condition is `elif path.suffix != ext:` but yours still works

File: create_termsheet.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    """
+    DESCRIPTION = (
+        "A command line tool to create a termsheet using "
+        "the Origin termsheet template with an Airbrush-compliant "
+        "json file"
+    )
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+    parser.add_argument(
+        ARGNAME_INPUT_JSON_PATH,
+        type=lambda f: validate_file_type(parser, f, ".json"),
```
Comment (spanishpy) [inline create_termsheet.py]: Send the format without the dot. Handle the comparison in the elif statement as `f".{ext}"`.

File: README.md [hunk 4]
```diff
@@ -13,3 +16,54 @@ The enclosed .yaml file leverages the OpenApi Specification, enabling consumers
...
+```sh
+git clone https://github.com/OriginMarkets/Airbrush.git
+cd Airbrush
+```
+
+Now that the repo is downloaded, get the project dependencies with the following
+[pip](https://www.python.org/downloads/) command:
+
+```sh
+pip install -r requirements.txt
```
Comment (uttamo) [inline README.md:94]: the convention would be to not make the user install these dependencies system-wide, rather, to install them inside a virtual environment. instructions for that look like:

Create and enter a virtual env
```sh
python3 -m venv env1
source env1/bin/activate
```

Install dependencies
```sh
pip install --upgrade pip
pip install -r requirements.txt
```

File: README.md (hunk 1 above)
Comment (johnjudeh) [reply README.md]: Thanks updated!

File: create_termsheet.py (hunk 2 above)
Comment (johnjudeh) [reply create_termsheet.py]: Makes sense, that's clearer. Will update

File: README.md (hunk 4 above)
Comment (johnjudeh) [reply README.md:94]: Yup makes sense. Was trying to keep the instructions simple but its not what I would do either.

File: create_termsheet.py (hunk 3 above)
Comment (johnjudeh) [reply create_termsheet.py]: Thanks, updated

File: README.md [hunk 5]
```diff
@@ -4,20 +4,61 @@
...
+the market.
+
+ 1. [Airbrush OpenAPI Specification](<Airbrush v1.0.yaml>): The Airbrush data specification defined in a
+ yaml file that leverages the [OpenApi](https://www.openapis.org/) Specification, enabling consumers to
+ import the data specification and generate a client in over 40 different languages including Java, Python
+ and Scala. The Origin platform publishes all trade data according to Airbrush; more information can be
+ found on the [Orign API documentation page](https://login2.originmarkets.com/api/trades/).
+
+ 2. [Machine-Readable Termsheet Template](<Machine-Readable Termsheet.docx>): A machine-readable termsheet
+ template created using the [Jinja](https://jinja.palletsprojects.com/en/3.0.x/) templating language. This
```
Comment (uttamo) [inline README.md]: really tiny thing but for the Jinja url I would use "https://jinja.palletsprojects.com/" just so it's a bit more general. When Jinja release newer versions, the general URL would automatically redirect to the latest version

Comment (johnjudeh) [reply README.md]: Thanks Uttam, updated this!

**Pull request merged**